backend/models/userModel.js

```diff
@@ -29,14 +30,45 @@ const userSchema = mongoose.Schema({
     },
   ],
   monster: [{
-      type: mongoose.Schema.Types.ObjectId,
-      ref: 'Monster'
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Monster'
   }],
   friends: [{
     type: mongoose.Schema.Types.ObjectId,
     ref: 'User'
-  }]
-
+  }],
+  registrationDate: {
+    type: Date,
+    default: Date.now,
+  },
+  // totalCompletedTodoLists: {
+  //   type: Number,
+  //   default: 0,
+  // },
+  // totalCompletedTasks: {
+  //   type: Number,
+  //   default: 0,
+  // },
```

Comment:
But I'm wondering if we really need that property? It might complicate our logic, and we could count it on the frontend with just one line of code.